Modules/AirCastingStyling/Sources/AirCastingStyling/Color.swift

```diff
@@ -48,7 +48,11 @@ extension Color {
         Color(red: 251/255, green: 253/255, blue: 255/255)
     }
     
-    public static var sessionCardShadow: Color {
+    public static var bottomCardShadow: Color {
         return Color(red: 205/255, green: 209/255, blue: 214/255, opacity: 0.36)
     }
+    
+    public static var sessionCardShadow: Color {
+        return Color(red: 205/255, green: 209/255, blue: 214/255)
+    }
```

Comment:
Maybe we could have something like 
```
public static var sessionCardShadow: sessionCardShadow2 {
    return sessionCardShadow2.values
}
}

public enum sessionCardShadow2 {
case values

var color: Color {
    switch self {
    default: return .red
    }
}

var radius: Int {
    switch self {
    default: return 0
    }
}
}
```
and then in every place which needs to use that, we could have `.shadow(color: .color, radius: .radius, x: .x, y: .y))` I am not sure if it should be enum or struct or sth else. In the end it is just a suggestion, please skip it if you don't want to consider 😎